esp/applications/eclqueries/jwt.yaml

```diff
@@ -0,0 +1,13 @@
+jwt:
+   objname: jwtsecmgr
+
+   ## Default properties
+   allowSelfSignedCert: false
+   keyFilePath: /opt/HPCCSystems/secrets/jwt-security/key
```

Comment:
It is a default value that does make sense, but it would perhaps make more sense to support the newer secrets vault as an option.

But note, this PR does not fully support containers just yet.  I'm target legacy configurations at the moment and hope to address container support Real Soon Now.

Reply:
The secrets code has been merged.
And you can use the secrets access code on baremetal as well.  The file just needs to be put in the right place.

Secrets are categorized you can use the system category for now.  
Either always use "jwt-security" as the secret name or configure it something like:
  secret: jwt-security


Then in bare metal put your key at /opt/HPCCSystems/secrets/system/jwt-security/key.

For both baremetal and k8s, in your code access the contents using:

If key is stored as a string:
StringBuffer s;
getSecretValue(s, "system", "jwt-security", "key", true);

Reply:
Tony, this implies that key location is not user-configurable, right?  Or are you expecting the code to parse a path (ominous grumblings at this point)?

Reply:
Well for my stuff going forward I hope to start treating all keys, certs, credentials, etc. as managed secrets.  The bare metal case is a little tricky.. not in how you find the secret, that just uses the jsecrets API, but in how the secret gets to the secret location in the first place.
Vaults can be used in k8s or baremetal, but again how it gets set up for baremetal is a question.
For k8s the secret would either be configured through k8s or stored in a vault and the configuration is pretty straight forward (I think).
Now the truth is if using secrets it would probably be best to combine everything to do with talking to and verifying tokens for a given jwt server under one secret with multiple child named values.. including the URL.  If you wanted to go that route you could look at my example for creating secrets for soapcall to use.

Reply:
That helps, thanks.

Reply:
This is done.  I took your advice and swapped 'secretsPath' in for 'keyFilePath'.  I know in the vault that doesn't make as much sense, but 'secrets' sound too much like actually defining the secret value rather than its location.